HPXMLtoOpenStudio/measure.rb

```diff
@@ -375,6 +375,15 @@ def self.check_for_errors()
         fail "There must be at least one floor adjacent to #{location}."
       end
     end
+
+    # Interior partition surfaces
+    (@hpxml.rim_joists + @hpxml.walls + @hpxml.foundation_walls + @hpxml.frame_floors).reverse_each do |surface|
+      next if surface.interior_adjacent_to.nil? || surface.exterior_adjacent_to.nil?
+
+      if [surface.interior_adjacent_to, surface.exterior_adjacent_to].sort == [HPXML::LocationLivingSpace, HPXML::LocationBasementConditioned].sort
+        fail "Interior partition surfaces should not be described using #{surface}."
```

Comment:
This feels unnecessary. Why would we allow InteriorAdjacentTo=living space and ExteriorAdjacentTo=living space, but not InteriorAdjacentTo=living space and ExteriorAdjacentTo=basement - conditioned? Or is there a nuance here I'm not appreciating?

Reply:
My intention was to prevent users from describing interior partitions using RimJoists, Walls, FoundationWalls, or FrameFloors. I wanted these constructions to not have conditioned space types (i.e. living space and basement - conditioned) as `interior_adjacent_to` and `exterior_adjacent_to` at the same time. 
However, as you pointed out, this wasn't doing what I intended 🤦 
Do you still think it's unnecessary to correct this part and prevent users from describing interior partitions?

Reply:
Hmm. I tend to feel that this is not necessary. I am pulling in the latest master and will remove this check.